src/graphics.rs

```diff
@@ -120,10 +122,10 @@ pub struct VarDisplay<'a> {
 impl<'a> VarDisplay<'a> {
     /// Create a new variable sized display.
     ///
-    /// Buffersize must be at least width / 8 * height bytes.
+    /// Buffersize must be at least (width + 7) / 8 * height bytes.
```

Comment:
Maybe add a small rounded up in brackets behind the line to make it easier to understand why the + 7 is necessary on a quick glance.

/// Buffersize must be at least (width + 7) / 8 * height bytes. (rounded up!)